db/data/regions/DE/en.yml

```diff
@@ -8,10 +8,12 @@ en:
             default: Street and house number
             optional: Street and house number (optional)
           errors:
-            blank: Enter a street and house number
+            blank_instructional: Enter a street and house number
+            blank_informative: Street and house number is missing
```

Comment:
Same comment as above.
If neither are present: Street and house number are missing
If one or the other is present: Street or house number is missing

Reply:
"Street and house number" are treated on this `address_1` field as a single field.

So if it's empty, both are missing.

Reply:
Got it – in that case we still need proper grammar for talking about 2 things, i.e. `Street and house number are missing`

Reply:
I will make that update! 👨‍🍳